test/engine/select.test.lua

```diff
@@ -148,3 +148,24 @@ s:fselect({}, {fselect_type='jira'})
 s:jselect()
 s:fselect({}, {fselect_widths={8, 8, nil, 10}})
 s:fselect(nil, {fselect_use_nbsp=false})
+s:drop()
+
+-- https://github.com/tarantool/tarantool/issues/6165
```

Comment:
Should not we place a bugfix test case into a new file like gh-xxxx-short-description.test.lua?